frontend/src/app/components/crm/crm.component.ts

```diff
@@ -49,9 +49,7 @@ export class CrmComponent implements OnInit {
 
   constructor(public titleService: Title, private loginService: LoginService, private webhookService: WebhookService) {
     this.loginService.getLoginResponse().subscribe((res: LoginResponse) => {
-      if (!res.loggedIn) {
-        location.href = this.logoutUrl;
-      } else {
+      if (res.loggedIn) {
         this.logoutUrl = res.url;
       }
```

Comment:
Are we no longer redirecting if they are not logged in or is this happening somewhere else now?

Reply:
We should use this instead: https://angular.io/guide/router#preventing-unauthorized-access